scalableminds/webknossos
# prevent tree tab from grabbing focus (#2627)

A tabIndex of 0 was assigned to the tree list by the react-virtualized component per default. Setting it to null prevents the list from grabbing the focus.

### Mailable description of changes:
 - [Fix] Switching to the trees tab no longer disables keyboard shortcuts.

### Steps to test:
- Click somewhere in the trees tab, the keyboard shortcuts (e.g. 1 to toggle visibility of all trees) should still work

### Issues:
- fixes #2627

------
